runtime/Cargo.toml

```diff
@@ -8,83 +8,128 @@ edition = "2018"
+cumulus-pallet-dmp-queue = { git = 'https://github.com/paritytech/cumulus', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-pallet-xcm = { git = 'https://github.com/paritytech/cumulus', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-pallet-xcmp-queue = { git = 'https://github.com/paritytech/cumulus', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-primitives-core = { git = 'https://github.com/paritytech/cumulus', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-primitives-timestamp = { git = 'https://github.com/paritytech/cumulus', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-primitives-utility = { git = 'https://github.com/paritytech/cumulus', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+parachain-info = { git = 'https://github.com/paritytech/cumulus', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
 
-[build-dependencies]
-substrate-wasm-builder =  { git = "https://github.com/paritytech/substrate", package = "substrate-wasm-builder" }
+# Polkadot Dependencies
+polkadot-parachain = { git = 'https://github.com/paritytech/polkadot', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+xcm = { git = 'https://github.com/paritytech/polkadot', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+xcm-builder = { git = 'https://github.com/paritytech/polkadot', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+xcm-executor = { git = 'https://github.com/paritytech/polkadot', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+pallet-xcm = { git = 'https://github.com/paritytech/polkadot', default-features = false , branch = "hc-contract-experiment-patched-v0.9.9" }
+
+# Contracts specific packages
+pallet-contracts = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts", default-features = false, branch = "cmichi-tmp-experiment" }
+pallet-contracts-primitives = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-primitives", default-features = false, branch = "cmichi-tmp-experiment" }
+pallet-contracts-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-rpc-runtime-api", default-features = false, branch = "cmichi-tmp-experiment" }
 
 [features]
 default = [
-    "std",
-    # temporarily enable unstable contracts features by default, remove this before using on a production chain.
-    "contracts-unstable-interface",
+	"std",
+	# temporarily enable unstable contracts features by default, remove this before using on a production chain.
+	"contracts-unstable-interface",
```

Comment:
I think we should remove it from the default set and document in the readme that users should supply the feature on the command line when they want to test unstable features. As debug message is stabilized it should be no issue to have a node without this feature. Only when they run a local testnet of course.